nuvation_nuv300_bms_driver/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(nuvation_nuv300_bms_driver)
+
+# add_compile_options(-std=c++11)
+
+set(catkin_RUN_PACKAGES
+	cob_msgs
+	roscpp
+)
+
+set(catkin_BUILD_PACKAGES
+	${catkin_RUN_PACKAGES}
+)
+
+find_package(catkin REQUIRED COMPONENTS
+	${catkin_BUILD_PACKAGES}
+)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/cmake")
+
+find_package(Modbus)
```

Comment:
why does Modbus needs to be found like this?

Reply:
Because it was not found else, I needed to create file nuvation_nuv300_bms_driver/cmake/FindModbus.cmake to make it work. Is there another option?

libmodbus-dev referenced in package.xml like you would refere to libopencv-dev or boost. It is installed with resdep install. So I imitated the opencv or boost inclusion processes here.